imops/measure.py

```diff
@@ -120,7 +120,7 @@ def label(
         res.append(('labels', np.array(range(1, num_components + 1))))
     if return_sizes:
         _, sizes = unique(labeled_image, return_counts=True)
-        res.append(('sizes', sizes[1:]))
+        res.append(('sizes', sizes[1:] if 0 in labeled_image else sizes))
```

Comment:
isn't this check super slow?

Reply:
No, e.g. for array of size `(512, 512, 512)` labeling took ~7 sec. while `0 in labeled_image` took ~0.1 sec.

Reply:
fair enough